src/components/Header/Header.tsx

```diff
@@ -24,9 +25,13 @@ const Header = React.forwardRef<HTMLDivElement, IHeader>((props, ref) => {
     >
       <div className="flex gap-1 items-center">
         {startAdornment}
-        <Link className="font-medium text-white" to="/">
-          {title}
-        </Link>
+        {to ? (
+          <Link className="font-medium text-white" to={to}>
+            {title}
+          </Link>
+        ) : (
+          <Link className="font-medium text-white" to="/">{title}</Link>
+        )} 
```

Comment:
Aqui poderia ser uma condicional mais simples para melhorar a legibilidade:
```suggestion
        <Link className="font-medium text-white" to={to || '/'}>
          {title}
        </Link>
```